spec/search_spec.rb

```diff
@@ -18,13 +34,25 @@
       expect(search_page.url_include?(data)).to be(true)
     end
 
-    it 'verifies user can not find any item by typing unexisted data' do
+    
+  end
+  
+  context 'when invalid credentials' do
+	it 'verifies user request will be described in the url field' do
       data = 'asdsadsada'
       current_url = browser.current_url
       search_page.search(data)
       search_page.wait_for{ current_url != browser.current_url }
       expect(search_page.url_include?(data)).to be(true)
     end
+	
+	it 'verifies that user request which consist unexisted data can not find anything' do
+      data = 'asdsadsada'
+      current_url = browser.current_url
+      search_page.search(data)
+      search_page.wait_for{ current_url != browser.current_url }
+      expect(search_page.results_label) == '0 results have been found.'
+    end
```

Comment:
seems few examples are same, the only changing thing is data. consider using shared examples or arrays\